CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # MicroProfile Language Server Changelog
 
+## [0.13.0](https://github.com/eclipse/lsp4mp/milestone/14?closed=1) (October 17, 2024)
+
+### Enhancements
+
+- feat: Provide property validator extension API. See [#460](https://github.com/eclipse/lsp4mp/pull/460).
+
+### Bug Fixes
+
+- REST endpoint codelens is not resolving a `public static final` parameter. See [#467](https://github.com/eclipse/lsp4mp/pull/467).
```

Comment:
Maybe something like "Codelens for REST endpoints should resolve variable references."

Reply:
fixed